app/ui/tweet/Comment.tsx

```diff
@@ -0,0 +1,78 @@
+import {Card, CardBody, CardHeader} from "@nextui-org/card";
+import useFetchComments from "@/app/ui/hooks/useFetchComments";
+import {Dispatch, SetStateAction, useCallback} from "react";
+import {Avatar} from "@nextui-org/avatar";
+import {Button, Input} from "@nextui-org/react";
+import {ITweet} from "@/app/lib/definitions";
+
+interface CommentProps {
+    tweet: ITweet,
+    setTweet: Dispatch<SetStateAction<ITweet | undefined>>
+}
+
+export default function Comment(commentProps: CommentProps) {
+
+    const {comments, comment} = useFetchComments(commentProps.tweet, commentProps.setTweet)
```

Comment:
Formatting could be a bit nicer (don't commit this suggestion right here, it's just for illustration)
```suggestion
    const {comments, comment} = useFetchComments(
         commentProps.tweet, 
         commentProps.setTweet
    );
```